images/retry.sh

```diff
@@ -8,13 +8,15 @@
 # and sleep & retry if there's a failure
 # $1 
 #	parameter contains the full command to be executed
-r=0
-retries=5
-while [ "$r" -le "$retries" ]
-do
-	echo "retry $r"
-	$1 && break
-	echo "error executing: $1"
-	r=$((r+1)) 
-	sleep 15
-done
\ No newline at end of file
+if [[ "$OSTYPE" == "linux-gnu" ]] || [[ "$OSTYPE" == "darwin"* ]]; then
```

Comment:
why are we checking linux type

Reply:
is it to bypass local windows dev situation

Reply:
Reverted. Please ignore